### ISSUE TITLE
Add pre-treatment column

### ISSUE THREAD

**Issue by gavinmcnicol** [International-Soil-Radiocarbon-Database/ISRaD#92]
It may be helpful to add controlled vocabulary to a new radiocarbon pre-treatment column in the layer tab, for sample pre-treatments performed as final step before radiocarbon analysis. For example, acid treatments to remove carbonates (e.g. used at University of Arizona). This is different to the frc_scheme which also includes schemes including use of acid, because the pre-treatment is not necessarily to isolate a particular fraction.

**Comment by jb388** [comment]
This sounds like a reasonable idea, but difficult to apply retroactively, i.e. to previously ingested templates. If we add this to the layer table we should probably also add it to fraction table. Open to more discussion.

**Comment by alkalifly** [comment]
I agree that it would be nice to include this information (it is definitely available from many of the original papers). And I don't think it should be added to the fraction table, since it would only be for treatments that get done to the *entire* layer, before bulk measurements are made, and before any layer fractionation takes place.

**Comment by kjmcfarlane** [comment]
Yes please! Bulk for sure. One column could at least give enough info to give folks a heads up that a pretreatment was used - if they care about the method they could go find it.It’s possible someone could do a fractionation with particle size and still pretreat the fractions before 14C - the silt and clay size fractions for example. I think it’s rare for people to pretreat density or chemical fractionation residues though.

**Comment by greymonroe** [comment]
Can you provide a list of controlled vocab for this proposed column? This list needs to be decided _definitely_ (can never be changed later) before the column can be added.

**Comment by kjmcfarlane** [comment]
I was thinking initially to make it a "radiocarbon pretreatment" column with "yes or blank" - like the composited sample question. But if you want to make it a pull down a complete list (as far as I can think of!) would be:
none
decanted acid only
decanted acid and base
acid fumigated
acid no removal
other

This is the most complete list I can think of. A shorter one could be:
none
acid only carbonate removal
other

I can give a description for the template info file if that is useful. A second opinion from someone at MPI or UCI that handles sample prep for 14C would be good if you can get one.

Karis

**Comment by kjmcfarlane** [comment]
"acid no removal" is kind of confusing. I mean that the sample acidified with liquid acid and the acid left in the sample. I can try to explain better if it's not clear!
